src/translations/product-list.translations.js

```diff
@@ -129,11 +129,40 @@ export const FILTER_BUTTON_TEXT = {
     value: 'Filter'
   }
 };
-
+export const PRODUCT_NOT_FOUND = {
+  0: {
+    lang: 'uk',
+    value: 'В цій категорії продуктів не знайдено'
+  },
+  1: {
+    lang: 'eng',
+    value: 'No products found in this product category'
+  }
+};
+export const PRICE_FROM = {
+  0: {
+    lang: 'uk',
+    value: 'від'
+  },
+  1: {
+    lang: 'eng',
+    value: 'from'
+  }
+};
+export const PRICE_TO = {
+  0: {
+    lang: 'uk',
```

Comment:
same

Reply:
I can't fix it now becouse it's configuration of all project. So that it should be change everywhere